packages/fxa-settings/src/components/Settings/BentoMenu/bento.svg

```diff
@@ -1,2 +1,2 @@
-<svg viewBox="0 0 24 24" fill="none" xmlns="http://www.w3.org/2000/svg"><rect width="6" height="6" rx="1" fill="#2F2247"/><rect x="9" width="6" height="6" rx="1" fill="#2F2247"/><rect x="18" width="6" height="6" rx="1" fill="#2F2247"/><rect y="9" width="6" height="6" rx="1" fill="#2F2247"/><rect x="9" y="9" width="6" height="6" rx="1" fill="#2F2247"/><rect x="18" y="9" width="6" height="6" rx="1" fill="#2F2247"/><rect y="18" width="6" height="6" rx="1" fill="#2F2247"/><rect x="9" y="18" width="6" height="6" rx="1" fill="#2F2247"/><rect x="18" y="18" width="6" height="6" rx="1" fill="#2F2247"/>
```

Comment:
Using `currentColor` instead of a hard coded colour allows Windows HCM to replace the icon color with the expected HCM color for buttons